examples/ml-pipelines-node-classification.ipynb

```diff
@@ -0,0 +1,1540 @@
+       "      <th>6</th>\n",
+       "      <td>6</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>7</td>\n",
+       "      <td>100</td>\n",
+       "      <td>100</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>8</td>\n",
+       "      <td>100</td>\n",
+       "      <td>100</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>9</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   nodeId  propertyValue  subject\n",
+       "0       0            100      100\n",
+       "1       1              1        1\n",
+       "2       2              2        2\n",
+       "3       3              2        2\n",
+       "4       4              3        3\n",
+       "5       5              3        3\n",
+       "6       6              4        4\n",
+       "7       7            100      100\n",
+       "8       8            100      100\n",
+       "9       9              4        4"
+      ]
+     },
+     "execution_count": 154,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Retrieve node information from Neo4j using the node IDs from the prediction result\n",
+    "nodes = gds.util.asNodes(predicted_fastrp.nodeId.to_list())\n",
+    "\n",
+    "# Create a new DataFrame containing node IDs along with node properties\n",
+    "nodes_df = pd.DataFrame([(node.id, node[\"subject\"]) for node in nodes], columns=[\"nodeId\", \"subject\"])\n",
+    "\n",
+    "# Merge with the prediction result on node IDs, to check the predicted value\n",
+    "# against the original subject\n",
+    "#\n",
+    "# NOTE: This could also be replaced by just appending `node[\"subject\"]` as a\n",
+    "# Series since the node order would not change, but a proper merge (or join)\n",
+    "# is clearer and less prone to errors.\n",
+    "predicted_fastrp.merge(nodes_df, on=\"nodeId\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Z27MnchLr9mi"
+   },
+   "source": [
+    "As we can see, the prediction for all the holdout nodes is accurate."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "UdEVypaiMc5d"
+   },
+   "source": [
+    "## Write result back to Neo4j"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "aKuUQwxbPOaV"
+   },
+   "outputs": [],
+   "source": [
+    "model_fastrp.predict_write(\n",
+    "    G,\n",
+    "    writeProperty=\"predictedSubject\",\n",
+    "    modelName=\"cora-pipeline-model-fastrp\",\n",
+    "    predictedProbabilityProperty=\"predictedProbabilities\",\n",
+    ")"
```

Comment:
I would like us to note, that you can also simple use `gds.graph.nodeProperties.write`. The current cell suggest you need to run `predict_write`.

Reply:
Do you mean that we can use `gds.graph.nodeProperties.write` because we've already mutated the graph? Or is it something else?